### DISCUSSION TITLE
What Autopilot keybinding with Thrustmaster Airbus Stick to disconnect autopilot?

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#6011] [Q&A] [answered]
hello
I was wondering what kind of command (what is the correct one) i should select in the FS menu for my Thrustmaster Airbus Stick to disconnect the AP (toggle AP master/disconnect AP/....etc etc etc...there are many options) 
Also the Autothrottle with my Thrustmaster Airbus Quadrant.

Thank you for some answers.

**Comment by tracernz** [comment, marked as answer]
One of these two for A/P, not sure which makes it work 😆 :
![image](https://user-images.githubusercontent.com/9995998/137269815-efd8eb02-1de3-4877-901d-4eb27da850b2.png)

And on the quadrant:
![image](https://user-images.githubusercontent.com/9995998/137270004-5dece649-836c-4d29-9729-5a0366f1c270.png)
